sandbox_externalkey_unix.go

```diff
@@ -60,7 +62,12 @@ func SetExternalKey(controllerID string, containerID string, key string) error {
 		ContainerID: containerID,
 		Key:         key}
 
-	c, err := net.Dial("unix", udsBase+controllerID+".sock")
+	execRoot := "/run/docker"
+	if v := os.Getenv("_LIBNETWORK_EXEC_ROOT"); v != "" {
```

Comment:
On second thought, maybe the underscore prefix is not needed?

Reply:
the original intention was to make sure this variable should not be set by human users

Reply:
Agree that the '_' is not necessary.  Doubt people will accidentally prefix an environment variable with `LIBNETWORK_`.

Reply:
removed the underscore prefix